content/_posts/2021-05-19-welcome-to-libera-chat.md

```diff
@@ -28,8 +28,8 @@ and we will never accept corporate control.
 
 Control of freenode infrastructure will soon be transferred to Freenode Limited
 and its agents. This means your data will soon be available to their personnel.
-We don't knows these people; neither do most of you. We can't claim that this
-is a good or even acceptable outcome, and are loath to entrust your data to a
```

Comment:
the 'are loath' is correct, if you want to use the verb form in english it's 'loathe' instead.

Reply:
Yeah, #2 fooled me into thinking it is wrong - that "fix" didn't seem right, though ;^)

Thanks.